README.md

```diff
@@ -97,18 +97,21 @@ Example `timeseries` request
 }
 ```
 
```

Comment:
I‘m not sure what you mean. While possible- what would be the use case? I find this rather confusing as documentaion?